tasks/dependencies.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  - name: create
+    description: Create the Minio, PostgreSQL, and Redis Dependency Zarf Packages
+    inputs:
+      options:
```

Comment:
Would be better to make this an explicit `architecture` instead

Reply:
The funny part is they are actually arch agnostic packages...